src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -15,70 +15,61 @@
 import net.sf.jabref.bibtex.InternalBibtexFields;
-import net.sf.jabref.model.EntryTypes;
+import net.sf.jabref.exporter.LatexFieldFormatter;
+import net.sf.jabref.external.WriteXMPEntryEditorAction;
+import net.sf.jabref.gui.*;
 import net.sf.jabref.gui.actions.Actions;
+import net.sf.jabref.gui.desktop.JabRefDesktop;
 import net.sf.jabref.gui.fieldeditors.*;
-import net.sf.jabref.gui.help.HelpFiles;
+import net.sf.jabref.gui.fieldeditors.TextArea;
+import net.sf.jabref.gui.fieldeditors.TextField;
 import net.sf.jabref.gui.help.HelpAction;
+import net.sf.jabref.gui.help.HelpFiles;
 import net.sf.jabref.gui.keyboard.KeyBinding;
 import net.sf.jabref.gui.menus.ChangeEntryTypeMenu;
-import net.sf.jabref.logic.autocompleter.AutoCompleter;
-import net.sf.jabref.exporter.LatexFieldFormatter;
-import net.sf.jabref.external.WriteXMPEntryEditorAction;
-import net.sf.jabref.gui.*;
-import net.sf.jabref.importer.fileformat.BibtexParser;
+import net.sf.jabref.gui.undo.*;
+import net.sf.jabref.gui.util.FocusRequester;
+import net.sf.jabref.gui.util.component.CheckBoxMessage;
+import net.sf.jabref.gui.util.component.VerticalLabelUI;
 import net.sf.jabref.importer.ParserResult;
+import net.sf.jabref.importer.fileformat.BibtexParser;
+import net.sf.jabref.logic.autocompleter.AutoCompleter;
 import net.sf.jabref.logic.l10n.Localization;
 import net.sf.jabref.logic.labelpattern.LabelPatternUtil;
 import net.sf.jabref.logic.search.SearchQueryHighlightListener;
 import net.sf.jabref.logic.util.date.TimeStamp;
+import net.sf.jabref.model.EntryTypes;
 import net.sf.jabref.model.database.BibDatabase;
 import net.sf.jabref.model.database.BibDatabaseMode;
-import net.sf.jabref.model.entry.*;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.EntryConverter;
+import net.sf.jabref.model.entry.EntryType;
+import net.sf.jabref.model.entry.TypedBibEntry;
 import net.sf.jabref.specialfields.SpecialFieldUpdateListener;
-import net.sf.jabref.gui.undo.NamedCompound;
-import net.sf.jabref.gui.undo.UndoableChangeType;
-import net.sf.jabref.gui.undo.UndoableFieldChange;
-import net.sf.jabref.gui.undo.UndoableKeyChange;
-import net.sf.jabref.gui.undo.UndoableRemoveEntry;
-import net.sf.jabref.gui.util.FocusRequester;
-import net.sf.jabref.gui.util.component.CheckBoxMessage;
-import net.sf.jabref.gui.util.component.VerticalLabelUI;
-import net.sf.jabref.gui.desktop.JabRefDesktop;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
+import javax.swing.*;
+import javax.swing.event.ChangeEvent;
+import javax.swing.event.ChangeListener;
+import javax.swing.text.JTextComponent;
+import java.awt.*;
+import java.awt.event.*;
+import java.beans.PropertyChangeEvent;
+import java.beans.VetoableChangeListener;
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.*;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
```

Comment:
I would recommend only using full qualified imports, easier to avoid conflicts, e.g. you import java.a* and google.com.b* and both contain different classes with the same name.